machine/machine.go

```diff
@@ -127,6 +130,60 @@ func parseCapacity(b []byte, r *regexp.Regexp) (uint64, error) {
 	return m * 1024, err
 }
 
+/* look for sysfs cpu path containing core_id */
+/* such as: sys/bus/cpu/devices/cpu0/topology/core_id */
+func getCoreIdFromCpuBus(cpuBusPath string, threadId int) (int, error) {
+	path := filepath.Join(cpuBusPath, fmt.Sprintf("cpu%d/topology", threadId))
+	file := filepath.Join(path, "core_id")
+
+	num, err := ioutil.ReadFile(file)
+	if err != nil {
+		// report threadId if no NUMA
```

Comment:
does this failure always indicate there is no NUMA?  Maybe only return threadId if the error is `os.IsNotExist(err)`.

Reply:
Done. I have removed it.

Reply:
I was suggesting 
```
if os.IsNotExist(err) {
    // report threadId if no NUMA
    return threadId, nil
} else if err != nil {
    return 0, err
}
```
I don't have enough context on the change to know if that is _actually_ the right thing to do here, but whatever you think the correct behavior is works for me.